### ISSUE TITLE
Tune the activity feed in the admin panel

### ISSUE THREAD

**Issue by pixdemo** [KeypDemoCorp/awesome-ui#622]
The API contract may need to be updated to support this.

Accessibility should be considered when implementing this.

This was flagged during the last round of QA testing.
